models/topics.models.js

```diff
@@ -16,3 +32,19 @@ exports.selectArticlesById = (article_id) => {
       return rows;
     });
 };
+
+exports.selectArticles = () => {
+  return db
+    .query(
+      `SELECT article_id,
+  title,
+  topic,
+  author,
+  created_at,
+  votes,
+  article_img_url FROM articles ORDER BY created_at DESC;`
+    )
+    .then((articles) => {
+      return articles;
+    });
```

Comment:
Another repeat? Perhaps a resolved merge conflict has left this in by mistake :)

Reply:
Yeah, again.
Deleted